.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: monthly
+  open-pull-requests-limit: 10
+  target-branch: master
```

Comment:
Please remove, this is pointless without actual dependencies other than the parent pom.

Reply:
+ the bom

But ok I will remove dependabot from the PR

Reply:
BOM basically defines versions for plugin dependencies which I would classify outside "actual" dependencies -- they don't affect what gets included, only some metadata (and in fact administering Jenkins becomes more annoying if plugins have needlessly recent dependencies IMO).